local-test-infra/local-partner-deployment.yaml

```diff
@@ -16,6 +16,7 @@ spec:
         test-network-function.com/generic: orchestrator
       annotations:
         test-network-function.com/defaultnetworkinterface: '"eth0"' # optional
+        k8s.v1.cni.cncf.io/networks: macvlan-conf # multus network
```

Comment:
what happens when we deploy this on ocp without the network definition yml?

Reply:
fixed, removed the line when deploying in OCP with mustache.